ceph_iscsi_config/lun.py

```diff
@@ -406,17 +406,18 @@ def unmap_lun(self, target_iqn):
             self.config.update_item("targets", target_iqn, target_config)
 
             # determine which host was the path owner
-            disk_owner = self.config.config['disks'][self.config_key]['owner']
```

Comment:
Nit: could just use `disk_owner = self.config.config['disks'][self.config_key].get('owner')`

Reply:
Thanks! already changed

Reply:
Thanks -- can you squash the two commits down into a single commit and re-push?

Reply:
@dillaman  Done, thanks!